src/resources/postcss/views/_month.pcss

```diff
@@ -4,4 +4,38 @@
  * @package TribeEventsCalendar
  *
  */
-.tribe-events {}
\ No newline at end of file
+.tribe-events {
+
+	.tribe-events-calendar-month__event-multiday {
```

Comment:
We're going to need various widths of this multiday component (1 day - 7 days). Let's add modifier classes to account for this.

Reply:
Yes, I was leaving that for the time we actually worked on that. The idea of this PR was to set a basic markup and styles for the multiday marker.